### PR TITLE
Lesson 0 Fixes

### PR DESCRIPTION
**Pull request by erikerikson**
Fix bugs
Add Windows commands and script

### CODE REVIEW

File: Lesson0_BeforeWorkshop/README.md [hunk 1]
```diff
@@ -33,54 +49,36 @@ IAM --> users --> select your user ID --> security credentials tab
...
-### Step 6: choose a unique $STAGE name and set the $REGION for your deployed services
-
-We recommend you use your LAN ID to ensure it's unique, but you can use any name you want, provided it does *not* use hyphens.  Go ahead and set a shell variable to use later on:
+Regardless, install the serverless.com deployment framework - this will make it easy to deploy serverless components to AWS
 
+#### OS X or Windows
 ```sh
-export STAGE=b0bb
-export REGION=us-west-2
+npm install -g serverless
```
Comment (laurwang) [inline Lesson0_BeforeWorkshop/README.md]: Are we going to lock to a particular version?

Comment (erikerikson) [inline Lesson0_BeforeWorkshop/README.md]: oh, right! yes

Comment (erikerikson) [inline Lesson0_BeforeWorkshop/README.md]: done!  thanks for noticing!

File: Lesson0_BeforeWorkshop/README.md [hunk 2]
```diff
@@ -82,3 +80,26 @@ npm install -g serverless
...
+### Step 5: Set Up Your Environment
+
+You will need to select a unique `STAGE` name and choose a `REGION` for your service to be deployed into
+
+We recommend you use your User ID to ensure it's unique and to allow you to search for it in the AWS console, but you can use any name you want.  Please define it as an environment variable for later use:
+
+#### OS X
+
+```sh
+export STAGE=b0bb
```
Comment (laurwang) [inline Lesson0_BeforeWorkshop/README.md]: You might want to preface this with: "Suppose your lanid is b0bb."

I found a stack called b0bb while cleaning up the stacks.

**Pull request merged**